panel.js

```diff
@@ -1249,7 +1249,11 @@ var dtpPanel = Utils.defineClass({
                 //gnome-shell < 3.30 needs an additional "screen" param
                 global.screen ? args.push(global.screen) : 0;
 
+                if (!Me.settings.get_boolean('scroll-panel-show-ws-popup'))
+                    Main.wm._workspaceSwitcherPopup = { display: () => {} };
                 Main.wm._showWorkspaceSwitcher.apply(Main.wm, args.concat([0, { get_name: () => 'switch---' + direction }]));
+                if (!Me.settings.get_boolean('scroll-panel-show-ws-popup'))
```

Comment:
You can remove this `if` and just set `Main.wm._workspaceSwitcherPopup = null;` even if we are showing the workspaces popup.

Reply:
This wouldn't prevent multiple instances of `WorkspaceSwitcherPopup` from being spawned, on multiple scrolls.
Low quality GIF, buy you should be able to notice transparency variation here:
![set popup to null without checking flag](https://user-images.githubusercontent.com/20030650/84667416-9dd5ba00-af22-11ea-8c7a-920427011e12.gif)

Reply:
Ha yes you're absolutely right... I just don't like the 2 consecutive `Me.settings.get_boolean('scroll-panel-show-ws-popup')` calls, can we set this boolean in a variable or something?

Reply:
:heavy_check_mark: done